src/components/ui/sidebar/nav-main.tsx

```diff
@@ -144,6 +199,7 @@ export function NavMain({ links }: { links: NavigationLink[] }) {
 
 function PopoverMenu({ link }: { link: NavigationLink }) {
   const [open, setOpen] = useState(false);
+  const Path = usePath();
```

Comment:
follow js naming conventions
```suggestion
  const path = usePath();
```